src/Components/Components/src/LayoutComponentBase.cs

```diff
@@ -16,6 +16,6 @@ public abstract class LayoutComponentBase : ComponentBase
         /// Gets the content to be rendered inside the layout.
         /// </summary>
         [Parameter]
-        public RenderFragment Body { get; set; }
+        public RenderFragment Body { get; set; } = default!;
```

Comment:
I'm a bit worried about this. I get the intention, but this typically happens within user code. Do we pass in an empty render fragment if a layout component is invoked without this?

Reply:
It would be nice if we could declare this as `RenderFragment?` to match reality, though it depends on how well `.razor` supports this.

If the user is still allowed to write `@Body` in their layout (and are not forced to write `@Body!`) then it would make sense for it to be declared as nullable. At runtime, if you evaluate `@someRenderFragmentVarThatIsActuallyNull` then it's not an error - it just doesn't render anything.